students/kylee/users/models.py

```diff
@@ -0,0 +1,12 @@
+from django.db            import models
+
+# Create your models here.
+class Users(models.Model) :
+    name = models.CharField(max_length=50)
+    email = models.EmailField(max_length=50, primary_key=True)
+    password = models.CharField(max_length=500)
+    tel = models.CharField(max_length=15)
+    birthday = models.DateField()
```

Comment:
1. 불필요한 주석은 지워주세요.
2. email에 unique 속성을 추가해주세요. 그렇지 않으면 사용자가 중복 저장될 수 있습니다.
3. tel -> `telephone` 변수를 줄여서 작성하지 않도록 주의해주세요.

```suggestion
class Users(models.Model) :
    name = models.CharField(max_length=50)
    email = models.EmailField(max_length=50, primary_key=True)
    password = models.CharField(max_length=500)
    telephone = models.CharField(max_length=15)
    birthday = models.DateField()
```

Reply:
수정사항 반영 후 커밋했습니다!

Reply:
생각해보니 이메일에 unique=true를 설정하면 pk=true를 할 이유가 없다고 판단하여
primary_key=true는 지우고 unique=true만 될 수 있게 재설정해서 커밋했습니다.
확인 부탁 드립니다.